<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIMInfo</class>
 <widget class="QWidget" name="AIMInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TabWidget3">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Profile</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>First Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="TextLabel2">
         <property name="text">
          <string>Last Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtFirst"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtLast"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="TextLabel3">
         <property name="text">
          <string>Middle:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QLineEdit" name="edtMiddle"/>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="TextLabel4">
         <property name="text">
          <string>Maiden:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="2">
        <widget class="QLineEdit" name="edtMaiden"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="TextLabel5">
         <property name="text">
          <string>Nick:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QLineEdit" name="edtNick"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="TextLabel6">
         <property name="text">
          <string>Street:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QLineEdit" name="edtStreet"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="TextLabel7">
         <property name="text">
          <string>City:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edtCity"/>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="TextLabel8">
         <property name="text">
          <string>State:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLineEdit" name="edtState"/>
       </item>
       <item row="5" column="4">
        <spacer name="Spacer4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="TextLabel9">
         <property name="text">
          <string>Zip:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edtZip"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="TextLabel10">
         <property name="text">
          <string>Country:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmbCountry"/>
         </item>
         <item>
          <spacer name="Spacer5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel1_2">
         <property name="text">
          <string>ScreenName:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="edtScreen"/>
       </item>
       <item row="10" column="0">
        <spacer name="Spacer6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Status</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="margin">
        <number>11</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edtClient"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="edtIntIP"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOnline">
         <property name="text">
          <string>Online:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtOnline"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="edtExtIP"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblExtIP">
         <property name="text">
          <string>External IP:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtNA">
         <property name="font">
          <font/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblClient">
         <property name="text">
          <string>Client:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel1_3">
         <property name="text">
          <string>Status:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbStatus"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblNA">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblIntIP">
         <property name="text">
          <string>Internal IP:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="Spacer4_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QPlainTextEdit" name="edtAutoReply"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TabWidget3</tabstop>
  <tabstop>edtScreen</tabstop>
  <tabstop>edtFirst</tabstop>
  <tabstop>edtLast</tabstop>
  <tabstop>edtMiddle</tabstop>
  <tabstop>edtMaiden</tabstop>
  <tabstop>edtNick</tabstop>
  <tabstop>edtStreet</tabstop>
  <tabstop>edtCity</tabstop>
  <tabstop>edtState</tabstop>
  <tabstop>edtZip</tabstop>
  <tabstop>cmbCountry</tabstop>
  <tabstop>edtClient</tabstop>
  <tabstop>edtIntIP</tabstop>
  <tabstop>edtOnline</tabstop>
  <tabstop>edtExtIP</tabstop>
  <tabstop>edtNA</tabstop>
  <tabstop>cmbStatus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
